staticcheck/doc.go

```diff
@@ -851,6 +851,15 @@ https://blog.digitalocean.com/how-to-efficiently-compare-strings-in-go/`,
 		Since: "2019.2",
 	},
 
+	"SA6006": {
+		Title: `Using io.WriteString to write bytes to a writer w`,
+		Text: `w.Write([]byte) should be used instead of io.writeString
+
+		string([]byte) allocates a buffer to create the string because of the
+		immutable nature of strings in go`,
+		Since: "2020.2",
```

Comment:
2020.2 has already been released and does not include this check. New, unreleased checks should say `Since: "Unreleased"` instead.